projects/packages/backup-helper-script-manager/CHANGELOG.md

```diff
@@ -0,0 +1,13 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## 0.1.0 - 2023-12-06
```

Comment:
Could you help me here by explaining how it all should be so that the "release" gets "released"?

Like (I would assume) most of A8C, I have no idea how that whole versioning/changelogger thing works/is supposed to work, so I just run `./tools/changelogger-validate-all.sh` repeatedly, and swear/cry/make random changes until it says that it's happy.

Reply:
Happy to.

For packages (or really any project that doesn't have `.extra.release-branch-prefix` set in its `composer.json`), a release happens when a commit is pushed to the `trunk` or `prerelease` branch where there are no non-dotfiles in the project's `/changelog/` directory.

More specifically, the lack of non-dotfiles means that no "x.y.z-alpha" line gets added to `CHANGELOG.md` when mirroring the build project to its mirror repo. The fact that the top entry in `CHANGELOG.md` is not a prerelease version number (e.g. `-alpha`) triggers a workflow in the mirror that creates a git tag for that version number. The creation of the git tag does two things: (1) Packagist picks up that tag as a release, and (2) another workflow in the mirror repo runs that creates an GitHub release for the version.

So, for the initial version of a package, you can do one of two things:
1. Leave `CHANGELOG.md` empty to start, just the header part (lines 1–6), along with a file in `.../changelog/` with an appropriate entry such as "Initial release". Later on when someone does a release of the package (probably to go along with a release of a plugin using the package) an entry will be created.
2. Create the entry in `CHANGELOG.md` and do not add any file in `.../changelog/`. That should result in all the workflows described above running to create the 0.1.0 tag.

Reply:
OK, thanks, I've purged `CHANGELOG.md`, and edited the entry in `changelog/` in 53a4299. Does it look right?

Reply:
That looks good to me!

Reply:
Thank you Brad!